domains/ot.is-local.org.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Orchid app",
+    "domain": "is-local.org",
+    "subdomain": "*.ot",
```

Comment:
If you are adding a `*` (wildcard) to your subdomain please make sure to change your file name, but also please give us a reason to why you need the wildcard subdomain?